components/sponsors/SponsorModal.vue

```diff
@@ -4,15 +4,15 @@
             <div class="lightBox__closeButton" @click="close">✕</div>
             <div class="lightBox__title">
                 <sponsor-card :logo-url="context.logo_url" small></sponsor-card>
-                <h1 class="text-xl mx-8 border-b-8 border-solid">
+                <h1 class="text-xl mx-8">
                     {{ context.name }}
                 </h1>
             </div>
-            <h2 class="text-xl text-yellow-500 mt-4">{{ $t('about') }}</h2>
+            <h2 class="lightBox__subtitle text-xl mt-6">{{ $t('about') }}</h2>
```

Comment:
According to the design spec (`38px`), plus leveraging Tailwind, the better margin would be `mt-7 (1.75rem)` => `35px`, as opposed to `mt-6 (1.5rem)` => `30px`
```suggestion
            <h2 class="lightBox__subtitle text-xl mt-7">{{ $t('about') }}</h2>
```

<img width="753" alt="figma screenshot" src="https://user-images.githubusercontent.com/7352613/124386472-3517d600-dd0d-11eb-8e3f-bcfe295a23c8.png">

Reply:
Thanks for the review.
An extended question: Should I move the tailwind class to the style block?

Reply:
I think, for better code readability, it's recommended to do so but not mandatory.